NEWS.md

```diff
@@ -1,3 +1,11 @@
+## 3.2.4 (Released)
+
+This release includes bug fixes related to marc21_withholdings metadataPrefix and incorrect number of records being returned from ListIdentifiers request.
+
```

Comment:
Add the compare log link, it will help while debugging something
`[Full Changelog](https://github.com/folio-org/mod-oai-pmh/compare/v3.2.3...v3.2.4)`